r2dbc/boot-r2dbc-notify-listen/src/main/java/com/example/demo/config/WebConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.example.demo.config;
+
+import static org.springframework.web.reactive.function.server.RouterFunctions.route;
+import static org.springframework.web.reactive.function.server.ServerResponse.noContent;
+
+import com.example.demo.listener.Notifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.server.RouterFunction;
+import org.springframework.web.reactive.function.server.ServerResponse;
+import reactor.core.publisher.Mono;
+
+@Configuration
+class WebConfig {
+
+    @Bean
+    public RouterFunction<ServerResponse> routes(Notifier notifier) {
+        return route().GET("/hello", request -> notifier.send()
```

Comment:
@coderabbitai, can you please help me to write Integration Test to check notify and listen